WORKSPACE

```diff
@@ -294,7 +294,7 @@ http_archive(
 
 # RapidJSON
 http_archive(
-    name = "rapidjson",
+    name = "com_github_tencent_rapidjson",
```

Comment:
In which instances should we use com_github_XYZ or parent_org_XYZ, and in which name of the project? Please add this into MR description.